<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TT Reclamations</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc1.qrc">
    <normaloff>:/resources/img/icontt2.png</normaloff>:/resources/img/icontt2.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1341</width>
     <height>631</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>35</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Profile</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>135</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>135</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nouveau Pass</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>135</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Comfirmer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Envoi (CA)</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>135</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_10"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>135</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>IP Incident</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>135</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="text">
         <string>Réclamation usuelle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>141</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ID/N_Ligne</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>141</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Produit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>141</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Catégorie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="text">
         <string>SAV DATA</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>141</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>UE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="text">
         <string>CNOC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>131</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Ouvert</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Résolu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>En attente</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>1061</width>
       <height>311</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Titre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP Incident</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Etat</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Produit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Catégorie</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Grp d'affectation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Motifs de transfert</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Solution</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Commentaire</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>1061</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3ca">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>301</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="rsc1.qrc">:/resources/img/ca.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Reception(CNOC)</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>1061</width>
       <height>371</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Titre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP Incident</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Etat</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Produit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Catégorie</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Grp d'affectation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Motifs de transfert</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Solution</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Commentaire</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_3cnoc">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>580</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>551</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="rsc1.qrc">:/resources/img/cnoc.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>UE</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>1061</width>
       <height>371</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Titre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP Incident</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Etat</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Produit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Catégorie</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Grp d'affectation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Motifs de transfert</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Solution</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Commentaire</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_3_ue">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>580</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>461</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="rsc1.qrc">:/resources/img/UE.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rsc1.qrc"/>
 </resources>
 <connections/>
</ui>
